doc/_read-me.txt

```diff
@@ -0,0 +1,14 @@
+Organization of these folders
+
+Th file  doc_xx.hs it is the configuration for java help by language.
```

Comment:
Typo: "Th" -> "The"

Better wording "For each translation, there is a Java help configuration file named doc_??.hs."